src/tests/test_utils/common_test_utils/include/common_test_utils/file_utils.hpp

```diff
@@ -162,7 +164,7 @@ inline void directoryFileListRecursive(const std::string& name, std::vector<std:
             if (entire->d_name == parent_dir || entire->d_name == current_dir) {
                 continue;
             }
-            std::string path = name + CommonTestUtils::FileSeparator + entire->d_name;
+            std::string path = name + FileSeparator + entire->d_name;
```

Comment:
anyway this file is required to clean up the duplicated method as in `ov::utils`